.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Device Request
+about: Request a new device to be supported.
+title: "Device name"
+labels: Device Request
+assignees: ''
+
+---
+
+<!--
+  - Before requesting a device, please make sure to search the open and closed issues for any requests in the past.
+  - Sometimes devices have been requested before but are not implemented yet due to various reasons.
+  - If there are no hits on your for the device, please proceed.
+  - If you're unsure if the device support is already requested, please ask for advise in our Discord chat: https://discord.gg/QFhTxqN
```

Comment:
If you're unsure if the device support is already requested > If you're unsure whether the device support was already requested